<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Room3</class>
 <widget class="QDialog" name="Room3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>kamer2</string>
  </property>
  <widget class="QPushButton" name="home">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="C">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>110</y>
     <width>121</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Â°C</string>
   </property>
  </widget>
  <widget class="QLabel" name="binnen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Binnen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="verlivht">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>120</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>kamer 3 lamp</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Tbuiten">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
  </widget>
  <widget class="QFrame" name="stat_verwarm">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="Tbinnen">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="verwarming">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>status verwarming</string>
   </property>
  </widget>
  <widget class="QLabel" name="buiten">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>260</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Buiten</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="temp">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>211</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QPushButton" name="verwarm">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>191</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>kamer 3 verwarming</string>
   </property>
  </widget>
  <widget class="QLabel" name="verlichting">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>280</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="text">
    <string>status verlichting</string>
   </property>
  </widget>
  <widget class="QFrame" name="stat_verlicht">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>330</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
